pai-management/doc/how-to-setup-dev-box.md

```diff
@@ -48,11 +48,11 @@ git clone https://github.com/Microsoft/pai.git
 cd pai/pai-management/
 
 # Build your dev-box.
-sudo docker build -t dev-box .
+sudo docker build -t dev-box
 
 ```
 
-### Start Your dev-box contianer
+### Start your dev-box container
 
 - Suppose the path of ```custom-hadoop-binary-path``` in your service-configuration is ```/pathHadoop```
 - Suppose the directory path of your cluster-configuration is ``/pathConfiguration````. Note: Don't change the configuration file name！
```

Comment:
syntax error in ``/pathConfiguration````

Reply:
I think it's better to use single backticks for inline code instead of three backticks according to [GitHub Guide](https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code).

And three backticks is only supported by [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/) for a code block while single backticks is supported in Markdown.